telemetry/definitions/commonDefinitions.json

```diff
@@ -773,7 +773,44 @@
             "name": "codecatalyst_devEnvironmentWorkflowStep",
             "type": "string",
             "description": "Workflow step name"
+        },
+        {
+            "name": "authConnectionsCount",
+            "type": "int",
+            "description": "Current number of authentication connections the user has."
+        },
+        {
+            "name": "authUIElement",
+            "type": "string",
+            "description": "Feature tab user is interacting in.",
+            "allowedValues": [
+                "awsResource",
+                "codewhisperer",
+                "codecatalyst"
+            ]
+        },
+        {
+            "name": "unlockedTabs",
+            "type": "string",
+            "description": "comma delimtied list of tabs that user already has valid connections to."
+        },
+        {
+            "name": "invalidInputFields",
+            "type": "string",
+            "description": "comma delimtied list of input fields that user has invalid inputs typed."
+
+        },
+        {
+            "name": "region",
```

Comment:
How is this different from `awsRegion` present today as a global argument?
https://github.com/aws/aws-toolkit-common/blob/f82e51208471921132e2b7a79c73116436ceb584/telemetry/telemetryformat.md?plain=1#L68

Reply:
this is input by user during authentication flow. 
do we capture that in global argument?

Reply:
We use awsRegion generically across the toolkit to indicate any action that requires a region specification